### DISCUSSION TITLE
What are your scores ?

### DISCUSSION THREAD

**Discussion by dilerbatu** [dscripka/openWakeWord#155] [General]
Hey everyone, I am trying to train a model that should detect "Hey Mizu". I have trained 50+ models in order to achieve good results but sometimes my models miss keywords. I think that it is related to my recall score. 

My best model's hyperparameters and scores are:
Hyperparameters:

Keyword: Hey Mizu
Number of samples: 50k
Negative weight: 5000
Step Size: 700k

Results:
Accuracy: 0.84
Recall: 0.68
FP rate: 5

Any ideas how can I improve my recall score ?

**Comment by dscripka** [comment]
This metrics look fairly good overall, and in my experience should lead to a reasonably well performing model (with perhaps a higher than desirable false-positive rate).

What are the conditions where you are seeing the model miss activations? Certain types of background noise, or certain ways of speaking the wake word?
